### PR TITLE
feat(organisations): create organisation card component

### PR DESCRIPTION
**Pull request by sniper365**
create organisation card component

### CODE REVIEW

File: pkg/app/src/pages/app/organisations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import Paper from '@mui/material/Paper'
+import Typography from '@mui/material/Typography'
+
+import { ItemList } from 'components/OrganisationCard/itemList'
+import { Layout } from 'src/layouts/default/layout'
+import { useBodiesQuery } from '@gamedao-haiku/graphql/dist'
+import { CircularProgress } from '@mui/material'
+
+export function OrganisationPage() {
+	const { loading, data } = useBodiesQuery()
```
Comment (DarkNebula0) [inline pkg/app/src/pages/app/organisations/index.tsx:12]: What happens when useBodiesQuery returns an error.
https://www.apollographql.com/docs/react/api/react/hooks/#usequery

File: pkg/app/src/components/OrganisationCard/item.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+
+import { fetchIpfsJson } from 'src/utils/ipfs'
+
+import { TileCard } from './modules/tileCard'
+
+export function Item({ data }) {
```
Comment (DarkNebula0) [inline pkg/app/src/components/OrganisationCard/item.tsx:7]: Missing parameter types(ComponentProps)

File: pkg/app/src/components/OrganisationCard/item.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	const [metadata, setMetadata] = useState<any>()
```
Comment (DarkNebula0) [inline pkg/app/src/components/OrganisationCard/item.tsx:8]: Do not use any, unless absolutely necessary. 
there is a folder @types where you can create an interface for the metadata.

File: pkg/app/src/components/OrganisationCard/itemList.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Grid } from '@mui/material'
+import { Item } from './item'
+
+export function ItemList({ data }) {
```
Comment (DarkNebula0) [inline pkg/app/src/components/OrganisationCard/itemList.tsx:4]: Missing parameter types(ComponentProps)

File: pkg/app/src/components/OrganisationCard/modules/tileCard.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+const toLink = '/app/organisations/'
+
+export const TileCard = (props) => {
+	const { selectedAccount } = useExtensionContext()
+	const theme = useTheme()
+	const bgPlain = { backgroundColor: theme.palette.grey[500_16] }
+	const title = props.data.name.length > 17 ? `${props.data.name.slice(0, 17)} ...` : props.data.name
+	const address = selectedAccount?.account.address
+
+	const SubHeader = () => {
```
Comment (DarkNebula0) [inline pkg/app/src/components/OrganisationCard/modules/tileCard.tsx:17]: Should be its own module.
Is inefficent, with each rerender the subhader is redefined.

**Review by DarkNebula0 — CHANGES_REQUESTED** [review]

**Comment by 2075** [comment]
@sniper365 please check https://www.conventionalcommits.org/en/v1.0.0/ as we rely on conventional commit messages to autogenerate changelogs.

**Comment by sniper365** [comment]
> 



> @sniper365 please check https://www.conventionalcommits.org/en/v1.0.0/ as we rely on conventional commit messages to autogenerate changelogs.

I will follow. Thanks.

**Comment by sniper365** [comment]
> 



> @sniper365 please check https://www.conventionalcommits.org/en/v1.0.0/ as we rely on conventional commit messages to autogenerate changelogs.

I will follow. Thanks.

**Pull request closed without merging**